parsers/PA.py

```diff
@@ -14,158 +14,173 @@
+        'BLM 4': 'coal',  # [7] Sheet "C-GE-1A-2 CapInstXEmp"
+        'BLM 5': 'oil',  # [7] Sheet "C-GE-1A-2 CapInstXEmp"
+        'BLM 6': 'oil',  # [7] Sheet "C-GE-1A-2 CapInstXEmp"
+        'BLM 8': 'oil',  # [7] Sheet "C-GE-1A-2 CapInstXEmp"
+        'BLM 9': 'oil',  # [7] Sheet "C-GE-1A-2 CapInstXEmp" mentions no fuel type, and given all other units are accounted for this must be the heat recovery boiler for the 3 diesel-fired units mentioned in [2]
+        'CADASA 1': 'biomass',  # [9]
+        'CADASA 2': 'biomass',  # [9]
+        'Cativá 1': 'oil',  # [1][2]
+        'Cativá 2': 'oil',  # [1][2]
+        'Cativá 3': 'oil',  # [1][2]
+        'Cativá 4': 'oil',  # [1][2]
+        'Cativá 5': 'oil',  # [1][2]
+        'Cativá 6': 'oil',  # [1][2]
+        'Cativá 7': 'oil',  # [1][2]
+        'Cativá 8': 'oil',  # [1][2]
+        'Cativá 9': 'oil',  # [1][2]
+        'Cativá 10': 'oil',  # [1][2]
+        'Cobre Panamá 1': 'coal',  # [3]
+        'Cobre Panamá 2': 'coal',  # [3]
+        'Costa Norte 1': 'gas',  # [4][5]
+        'Costa Norte 2': 'gas',  # [4][5]
+        'Costa Norte 3': 'gas',  # [4][5]
+        'Costa Norte 4': 'gas',  # [4][5]
+        'Esperanza 1': 'oil',  # [7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
+        'Esperanza 2': 'oil',  # [7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
+        'Esperanza 3': 'oil',  # [7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
+        'Esperanza 4': 'oil',  # [7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
+        'Esperanza 5': 'oil',  # [7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
+        'Esperanza 6': 'oil',  # [7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
+        'Esperanza 7': 'oil',  # [7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
+        'Jinro': 'oil',  # [6][7]
+        'Pacora 1': 'oil',  # [6]
+        'Pacora 2': 'oil',  # [6]
+        'Pacora 3': 'oil',  # [6]
+        'PanAm 1': 'oil',  # [6][7]
+        'PanAm 2': 'oil',  # [6][7]
+        'PanAm 3': 'oil',  # [6][7]
+        'PanAm 4': 'oil',  # [6][7]
+        'PanAm 5': 'oil',  # [6][7]
+        'PanAm 6': 'oil',  # [6][7]
+        'PanAm 7': 'oil',  # [6][7]
+        'PanAm 8': 'oil',  # [6][7]
+        'PanAm 9': 'oil',  # [6][7]
+        'Sparkle Power 1': 'oil',  # [10]
+        'Sparkle Power 2': 'oil',  # [10]
+        'Sparkle Power 3': 'oil',  # [10]
+        'Sparkle Power 4': 'oil',  # [10]
+        'Sparkle Power 5': 'oil',  # [10]
+        'Sparkle Power 6': 'oil',  # [10]
+        'Sparkle Power 7': 'oil',  # [10]
+        'Sparkle Power 8': 'oil',  # [10]
```

Comment:
Similar to above, this is the Sparkle Power mapping, which is more straightforward.